scripts/create_perf_json.py

```diff
@@ -276,10 +276,11 @@ def fix_name(name: str) -> str:
 
     def __init__(self, shortname: str, unit: str, jd: Dict[str, str]):
         """Constructor passed the dictionary of parsed json values."""
-        def get(key: str) -> str:
+        def get(key: str, counter: bool = False) -> str:
             drop_keys = {'0', '0x0', '0x00', 'na', 'null', 'tbd'}
             result = jd.get(key)
-            if not result or result in drop_keys:
+            # For the Counter field, value '0' is reasonable
+            if not result or (result in drop_keys and not counter):
```

Comment:
Can we base this logic off of `key` instead of adding a new input for one key? Such as  `and key != 'Counter')`?

Reply:
Yes, this is a much better solution than the original change. I just updated the code. Thanks!